src/components/screens/map-screen/map-screen.tsx

```diff
@@ -275,6 +275,50 @@ class MapScreen extends React.Component<Props, State> {
     this.setState({ loadingUserSettings: false });
   }
 
+  /**
+   * Maps favourite locations to locations 
+   */
+  private mapLocationsFromFavouriteLocations = (favouriteLocations:FavouriteLocation[]):Location[] => {
+    return favouriteLocations.map((element) => {
+      const name = element.name;
+      const coordinates = element.latitude + "," + element.longitude;
+      return { name, coordinates };
+    }).slice(
+      0,
+      NUMBER_OF_RESULTS_FOR_FAVOURITE_PLACES
+    );
+  }
+
+  /**
+   * Filters favourite locations with a specific keyword
+   * @param keyword the string to search
```

Comment:
```suggestion
   * Filters favourite locations with a specific keyword
   *
   * @param keyword the string to search
```
Empty line needed between description and other things